app/configuration/Config.scala

```diff
@@ -104,12 +99,6 @@ object Config {
     }
   }
 
-  lazy val casService = {
```

Comment:
this has moved to somewhere that deserves an execution context.  Cas"Service" isnt exactly configuration

Reply:
👍